src/Components/SmartComponents/Reports/CustomReportFilter.js

```diff
@@ -85,6 +85,7 @@ const CustomReportFilter = ({ filterName, filterData, setFilterData }) => {
     const filterRadios = (options, filterName) => (
         <Select
             variant="single"
+            ouiaId="filterRadios"
```

Comment:
I would prefer to use `ouiaId` parameter for `filterRadios` and use it in `filterContent`
```js
return filterRadios(PUBLIC_DATE_OPTIONS, 'publish_date', 'publish-date');
```